core/crypto/crypto_store/crypto_store_impl.cpp

```diff
@@ -238,8 +238,7 @@ namespace kagome::crypto {
     return ed25519KeyToLibp2pKeypair(kp);
   }
 
-  libp2p::crypto::KeyPair CryptoStoreImpl::ed25519KeyToLibp2pKeypair(
-      const Ed25519Keypair &kp) const {
+  libp2p::crypto::KeyPair ed25519KeyToLibp2pKeypair(const Ed25519Keypair &kp) {
```

Comment:
Why is constantness removed?

Reply:
Because it's not class member, but free function.